src/bundle/Resources/public/js/CKEditor/custom-styles/inline/custom-style-inline-ui.js

```diff
@@ -38,17 +40,28 @@ class IbexaCustomStyleInlineUI extends Plugin {
     init() {
         this.editor.ui.componentFactory.add('ibexaCustomStyleInline', (locale) => {
             const dropdownView = createDropdown(locale);
-            const customStylesInline = Object.entries(window.ibexa.richText.customStyles).filter(([, config]) => config.inline);
+            const { customStyles } = ibexa.richText;
+            const customStylesInline = Object.entries(customStyles).filter(([, config]) => config.inline);
             const customStylesButtons = customStylesInline.map(this.createButton);
+            const defaultLabel = Translator.trans(/*@Desc("Custom styles")*/ 'custom_styles_btn.label', {}, 'ck_editor');
```

Comment:
translation extract?

Reply:
this is an old translation, it was only assigned to a variable.